### PR TITLE
Plan Selection Screen, Forgot Password Screen (1st, 2nd, 3rd), OTP Timer, Sign In Screen Validation, Sign Up Screen (1st), Fix OTP Input,

### PR DESCRIPTION
**Pull request by SPK137**
- Plan Selection Screen
image
image
- User Interface Forgot Password Screen (1st, 2nd, 3rd) (Need to connect to backend)
image
image
image
image
- OTP Timer
image
- Sign In Screen Validation (Need to connect to backend)
image
- Sign Up Screen (1st) (Need to connect to backend)
image
image
- Fix OTP Input (Can be accessed with `react-hook-form` now)

### CODE REVIEW

File: src/components/molecules/PlanSelectionCard.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        mb={5}
+        w="full">
+        <Image
+          source={image}
+          alt="Plan Selection Image"
+          width={150}
+          height={150}
+          resizeMode="contain"
+          mr={3}
+          // width={ScreenWidth}
```
Comment (pointmekin) [inline src/components/molecules/PlanSelectionCard.tsx]: Still using this?

File: src/screens/ForgotPasswordScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  third = 3
+}
+
+const ForgotPasswordScreen = () => {
+  const {
+    control,
+    formState: { errors },
+    watch
+  } = useForm();
+  // const { t } = useTranslation();
```
Comment (pointmekin) [inline src/screens/ForgotPasswordScreen.tsx]: Consider removing commented code?

File: src/screens/SignInScreen.tsx [hunk 3]
```diff
@@ -45,28 +52,57 @@ const SignInScreen = () => {
...
+              placeholder={t('profile.phoneNumber')}
+              name="phoneNumber"
+              keyboardType="number-pad"
+              control={control}
+              errors={errors}
+              isRequired
+              errorMessage={t('error.isRequired').replace(
+                '${name}',
+                t('profile.phoneNumber')
+              )}
```
Comment (pointmekin) [inline src/screens/SignInScreen.tsx]: you can do this with i18n.

```
// en.json
"key": "hello, {{ name }}",

// jsx
t("key", { name: "John Doe" })
```

File: src/screens/SignInScreen.tsx [hunk 4]
```diff
@@ -45,28 +52,57 @@ const SignInScreen = () => {
...
+              label={t('auth.password')}
+              placeholder={t('auth.password')}
+              name="password"
+              control={control}
+              errors={errors}
+              isRequired
+              errorMessage={t('error.isRequired').replace(
+                '${name}',
+                t('auth.password')
+              )}
```
Comment (pointmekin) [inline src/screens/SignInScreen.tsx]: same here

File: src/screens/SignUpScreen.tsx [hunk 5]
```diff
@@ -1,50 +1,181 @@
...
+                  placeholder={t(info.placeholder || info.label)}
+                  name={info.name}
+                  control={control}
+                  errors={errors}
+                  hasRequiredStar
+                  isRequired
+                  errorMessage={t('error.isRequired').replace(
+                    '${name}',
+                    t(info.label)
+                  )}
```
Comment (pointmekin) [inline src/screens/SignUpScreen.tsx]: same here

File: src/screens/SignUpScreen.tsx [hunk 6]
```diff
@@ -1,50 +1,181 @@
...
```
Comment (pointmekin) [inline src/screens/SignUpScreen.tsx]: same here

File: src/screens/SignInScreen.tsx (hunk 3 above)
Comment (SPK137) [reply src/screens/SignInScreen.tsx]: Fixed!

File: src/screens/SignUpScreen.tsx (hunk 5 above)
Comment (SPK137) [reply src/screens/SignUpScreen.tsx]: Fixed!

File: src/screens/SignInScreen.tsx (hunk 4 above)
Comment (SPK137) [reply src/screens/SignInScreen.tsx]: Fixed!

File: src/screens/SignUpScreen.tsx (hunk 6 above)
Comment (SPK137) [reply src/screens/SignUpScreen.tsx]: Fixed

File: src/components/molecules/PlanSelectionCard.tsx (hunk 1 above)
Comment (SPK137) [reply src/components/molecules/PlanSelectionCard.tsx]: comment removed

File: src/screens/ForgotPasswordScreen.tsx (hunk 2 above)
Comment (SPK137) [reply src/screens/ForgotPasswordScreen.tsx]: Removed!

**Review by pointmekin — APPROVED** [review]

**Pull request merged**